projects/sound-open-firmware/fetch-sdk.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eu
+
+# Zephyr doesn't provide a "latest" link, so clone the source tree
+# that produces the SDK (much smaller than the tarball itself, so
+# minimal overhead) and find the latest version tag as a proxy.  Will
+# likely break if the script is run in the moments between tagging a
+# release and the tarball appearing on github, but the risk is low and
+# it will recover with a retry.
+
+git clone --filter=tree:0 https://github.com/zephyrproject-rtos/sdk-ng
+
+VER=$(git -C sdk-ng tag -l 'v*' | sort -rV | head -1 | sed 's/v//')
```

Comment:
Do we have any concerns about hitting a major version automatically?

Reply:
With the SDK, the risk is low.  In fact it's been on version "zero" for like a decade. :)  It's really just a bunch of sdk-ng toolchains, in this case only the x86_64 one.  And even that is mostly degenerate as the **actual** compiler used to build the fuzzers is a host-provided clang anyway.  We need it because the Zephyr build still uses a lot of the binutils stuff for auxilliary steps and isn't yet 100% llvm-pure.